### ISSUE TITLE
Cannot detect any objects in images after trainning 

### ISSUE THREAD

**Issue by Aaron4Fun** [jianing-sun/Food-Detection-by-YOLOv2-with-Transfer-Learning#1]
Hello Sir, Thanks so much for your effort. Have you met a situation where the validation loss is higher than the training loss during the training process? Is that normal? I got an outcome with 0.11 train_loss and 0.08 val_loss, but the model still cannot detect any objects when I used it in the training dataset. Are there any things wrong? Any suggestion will be appreciated.

**Comment by jianing-sun** [comment]
Hi, 
Yes, it is normal if the validation loss is higher than the training loss, as there's always over-fitting on the training set. 
The information you offered is too general to tell exactly where was wrong. It could be a lot of reasons why it didn't work, depends on how you implemented. I would suggest you to check your code again.

**Comment by Aaron4Fun** [comment]
Sorry, train_loss is higher than the val_loss 
![image](https://user-images.githubusercontent.com/42711020/52539340-f916fb80-2d74-11e9-9450-da400ab75710.png)

**Comment by Aaron4Fun** [comment]
I think it is nice because both losses are close to 0, but it still cannot detect anything in the training dataset.

**Comment by Aaron4Fun** [comment]
![image](https://user-images.githubusercontent.com/42711020/52539419-ccafaf00-2d75-11e9-8fc6-2a3841fb9f4e.png)

**Comment by jianing-sun** [comment]
> I think it is nice because both losses are close to 0, but it still cannot detect anything in the training dataset.

Well, it really depends on how you compute the loss and the output of the network. Just to debug your code, like visualize or print the components of loss during training.

**Comment by jianing-sun** [comment]
IMO, the loss couldn't start with almost zero in the first epoch. So I think your code is somewhat wrong.

**Comment by Aaron4Fun** [comment]
That is the final result I got from 100 epochs. I use the same loss function as you are. Have you met the same problem as this?

**Comment by jianing-sun** [comment]
No
